mmpy_bot/plugins/example.py

```diff
@@ -28,14 +30,20 @@ async def busy_reply(self, message: Message):
             f"Number of busy worker threads: {busy}",
         )
 
-    @listen_to("hello_click", needs_mention=True)
+    @listen_to(
+        "hello_click",
+        needs_mention=True,
+        category="click",
+        syntax="hello_click --keyword-arg --flag",
```

Comment:
What does this `syntax` argument do? We should be able to obtain this information from `click` rather than having to hardcode it?